doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #TODO: start using the secret store.
</pre><p>remove this line.</p><p>also, i cant find a ticket for implementing this step. can you maybe show me the link?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>@ahsimb we try to avoid #TODO comments in code and instead create an issue and link it to the epic</p>